src/CONST.ts

```diff
@@ -3299,6 +3299,8 @@ const CONST = {
         DEFAULT: 'default',
     },
     SUBSCRIPTION: {
+        NEW_PRICING_START_DATE: new Date(2025, 3, 1),
+        NEW_PRICE_2025: 'newPrice2025',
```

Comment:
My apologies, I should've made a better suggestion. Let's use the same naming as we did in the backend / OldDot frontend: `PRICING_TYPE_2025: 'team2025Pricing'